mono/metadata/boehm-gc.c

```diff
@@ -1721,6 +1721,19 @@ mono_gc_ephemeron_array_add (MonoObject *obj)
 	return TRUE;
 }
 
+void
+mono_gc_get_gcmemoryinfo (gint64* high_memory_load_threshold_bytes,
+						  gint64* memory_load_bytes,
+						  gint64* total_available_memory_bytes,
+						  gint64* heap_size_bytes,
+						  gint64* fragmented_bytes)
+{
+	high_memory_load_threshold_bytes = 0L;
```

Comment:
`*high_memory_load_threshold_bytes = 0L`?

Reply:
Right. ‘0’ suffices, no need for ‘0L’.

Reply:
And yeah need a star.
Yes star.
No L.